01week/rockPaperScissors.js

```diff
@@ -12,6 +12,62 @@ function rockPaperScissors(hand1, hand2) {
 
   // Write code here
 
+  //First will convert the input to lowercase. Then will check if the input is correct ( rock, paper, scissors). 
+  // If the input is not correct the program won't run and it'll show "Wrong input! Please enter rock,paper or scissors."
+  //Next will compare the two inputs and show the winner.
+  
+  hand1= hand1.toLowerCase().trim();
+  hand2= hand2.toLowerCase().trim();
```

Comment:
In functional programming you want to avoid premaritally changing given parameters. A better practice would be to create new variables. 
```javascript
const  hand1Formatted = hand1.toLowerCase().trim();
```